cdap-app-templates/cdap-etl/cdap-etl-api/pom.xml

```diff
@@ -35,6 +35,11 @@
       <groupId>co.cask.cdap</groupId>
       <artifactId>cdap-api</artifactId>
     </dependency>
+    <dependency>
+      <groupId>commons-lang</groupId>
+      <artifactId>commons-lang</artifactId>
```

Comment:
Because classes in api module are for plugin to implement. The more dependencies it has, the higher chance there will be conflict in library version/classes.